src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem';
+
+type Props = {
+  items: Todo[],
+};
+
+export const TodoList: React.FC<Props> = ({
+  items,
+}) => {
+  return (
+    <ul className="todo-list" data-cy="todosList">
```

Comment:
you can't pass the tests because of this `ul`. already answered about it in chat